tslib.js

```diff
@@ -40,6 +40,12 @@ var __asyncValues;
         factory(createExporter(root));
     }
     function createExporter(exports, previous) {
+        if (typeof Object.defineProperty === "function") {

```

Comment:
You have to feature test using Object.create. Object.defineProperty existed in IE8 but only supported DOM elements. However, IE8 did not support Object.create.